examples/armv4t_multicore/emu.rs

```diff
@@ -207,17 +213,20 @@ impl Emu {
         // valid, and you should track which specific TID the GDB client requested to be
         // single-stepped, and run them appropriately.
```

Comment:
would it be safe to remove this old code comment now that you've added the ability to only step a single core at a time?